iocage_lib/ioc_start.py

```diff
@@ -1245,3 +1305,160 @@ def find_bridge_mtu(self, bridge):
         ).split()
 
         return membermtu[5]
+
+    def __check_nat__(self, backend='ipfw'):
+        su.run(
+            ['sysctl', '-q', 'net.inet.ip.forwarding=1'], capture_output=True
+        )
+
+        if backend == 'pf':
+            self.__check_nat_pf__()
+        else:
+            self.__check_nat_ipfw__()
+
+    def __check_nat_pf__(self):
+        su.run(['kldload', '-n', 'pf'])
+        pfctl = su.run(['pfctl', '-e'], capture_output=True)
+
+        if pfctl.returncode != 0:
+            if 'enabled' not in pfctl.stderr.decode():
+                iocage_lib.ioc_common.logit({
+                    'level': 'EXCEPTION',
+                    'message': pfctl.stderr.decode()
+                }, _callback=self.callback,
+                    silent=self.silent,
+                    exception=ioc_exceptions.CommandFailed)
+
+    def __check_nat_ipfw__(self):
+        su.run(
+            ['sysctl', '-q', 'net.inet.ip.fw.enable=1'], capture_output=True
+        )
+        su.run(['kldload', '-n', 'ipfw'])
+
+    def __add_nat__(self, nat_interface, forwards, backend='ipfw'):
+        if backend == 'pf':
+            self.__add_nat_pf__(nat_interface, forwards)
+            pf = su.run(
+                ['pfctl', '-f', '/tmp/iocage_nat_pf.conf'], capture_output=True
```

Comment:
you should probably return `/tmp/iocage_nat_pf.conf` as a value from `__add_nat_pf__` to not repeat yourself. Same for ipfw